omod/src/main/webapp/pages/forms/extension.gsp

```diff
@@ -67,13 +67,13 @@ ${ui.message("formentryapp." + extensionForm.uiLocation)}
 </p>
 
 <p><label name="extensionForm.order">${ui.message("formentryapp.order")}</label>
-<input type="number" id="extensionForm.order" name="extensionForm.order" value="${extensionForm.order}" required/></p>
+<input type="number" id="extensionForm.order" name="extensionForm.order" value="${ui.escapeAttribute(extensionForm.order)}" required/></p>
```

Comment:
Is this necessary for a numeric value like order?

Reply:
I'd say yes if order is a user entered value. Just because although the numeric field might limit the type of characters that can be entered in through the ui, a post request can still be used to inject JS and what not(this is actually the case for https://issues.openmrs.org/browse/RA-1317). I'd escape it just to be on the safer side.